namespaces/live-1.cloud-platform.service.justice.gov.uk/cp-dummy-gitops-dev/00-namespace.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cp-dummy-gitops-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "dev"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "cloud-platform-non-admin"
+    cloud-platform.justice.gov.uk/application: "helloworld-ruby"
+    cloud-platform.justice.gov.uk/owner: "cloud-platform-non-admin: "
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/cloud-platform-helloworld-ruby-app-gitops"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: concourse-cloud-platform-non-admin
```

Comment:
This additional namespace here `concourse-cloud-platform-non-admin` is needed?

Reply:
This is for the concourse pipeline used by team